elements/lisk-tree/src/sparse_merkle_tree/branch_node.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Leaf } from './leaf_node';
+import { branchHash } from './utils';
+
+export class Branch {
+	private _left: Leaf;
+	private _right: Leaf;
+	private _hash: Buffer;
+	public constructor(left: Leaf, right: Leaf) {
+		this._left = left;
+		this._right = right;
+		this._hash = branchHash(this._left.hash, this._right.hash);
+	}
+
+	public get hash() {
+		return this._hash;
+	}
+	public get left() {
+		return this._left;
+	}
+	public get right() {
+		return this._right;
+	}
+	public update(newChild: Leaf, side: boolean) {
```

Comment:
Use of `side` is not readable. 
```suggestion
	public update(newChild: Leaf, left: boolean) {
```